api/deploy/kubernetes/nfs/csidriver.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: "{{ .Name }}"
+spec:
+  attachRequired: false
+  volumeLifecycleModes:
+    - Persistent
+  fsGroupPolicy: File
```

Comment:
```suggestion
```

match with https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/v3.1.0/csi-nfs-driverinfo.yaml ?

Reply:
Hmm, not sure where that came from. Will remove it.